src/store.js

```diff
@@ -0,0 +1,9 @@
+import thunk from "redux-thunk" 
+import { createStore, applyMiddleware } from "redux";
+
+// Reducer
+import TodoListReducer from "./pages/todoSetup/reducers";
+
+const store = createStore(TodoListReducer, applyMiddleware(thunk));
```

Comment:
As per createStore contract, second argument is actually an initial state and we are passing enhancer instead.
Please check if it is working as expected.
Please refer : https://redux.js.org/api/createstore

Reply:
@Sg1211  This still works as createStore contract has two different interfaces which allows it to accept either reducer and optional enhancer OR reducer , optional initial state and optional enhancer